peer_review_data/main.py

```diff
@@ -12,50 +12,47 @@
 LOGGER: Logger = getLogger(__name__)
 
 
-def saveCourseAndUsers(canvasCourse: CanvasCourse) -> (bool, Course):
+def saveCourseAndUsers(canvasCourse: CanvasCourse) -> Course:
     course = models.Course.fromCanvasCourse(canvasCourse)
     LOGGER.info(f'Saving {course}…')
     course.save()
 
     LOGGER.info(f'Saving users of {course}…')
-    for canvasUser in canvasCourse.get_users():
+    '''
+    Possible bug in `canvasapi` module…
+    When calling `get_users` with the argument `include='test_student'`, the
+    module adds it to the URL as `include=test_studentanalytics_url`.  The
+    origin of the "analytics_url" string is unknown.  However, it's known that
+    the Canvas API expects the "include" argument to appear as `include[]`, so
+    by using a dictionary here with the key in the correct format, the bug is
+    bypassed and the argument appears correctly.
+    Strangely, a later call to `get_rubric` uses the `include` keyword argument
+    without problem.
+    '''
+    for canvasUser in canvasCourse.get_users(**{'include[]': 'test_student'}):
         try:
             user = models.User.fromCanvasUser(canvasUser)
             LOGGER.debug(f'Saving {user}…')
             user.save()
         except Exception as e:
             LOGGER.warning(f'Error saving {user}: {e}')
 
-    return True, course
+    return course
 
 
 def saveSubmissions(canvasAssignment: CanvasAssignment):
     canvasSubmissions: List[CanvasSubmission] = \
         canvasAssignment.get_submissions()
 
     for canvasSubmission in canvasSubmissions:
-        '''
-        The only untyped submission found during development was from
-        "Test Student".  It was discovered because that student wasn't
-        included with the rest of the course users and the relationship
-        between submission and missing user couldn't be created.  Testing
-        for untyped submissions eliminates those from "Test Student" and
-        possibly others.
-        '''
-        if canvasSubmission.submission_type is None:
-            LOGGER.warning('Skipping untyped submission '
-                           f'({canvasSubmission.id}) for '
-                           f'user ID ({canvasSubmission.user_id}).')
-            continue
-
```

Comment:
I guess this is why the test student is now included. Hm...

Reply:
Will resolve later, in #12/